src/v2/guide/components.md

```diff
@@ -963,68 +963,57 @@ The content distribution API is a very useful mechanism when designing component
 
 > New in 2.1.0+
 
-A scoped slot is a special type of slot that functions as a reusable template (that can be passed data to) instead of already-rendered-elements.
-
-In a child component, pass data into a slot as if you are passing props to a component:
-
-``` html
-<div class="child">
-  <slot text="hello from child"></slot>
-</div>
-```
-
-In the parent, a `<template>` element with a special attribute `slot-scope` must exist, indicating that it is a template for a scoped slot. The value of `slot-scope` will be used as the name of a temporary variable that holds the props object passed from the child:
-
-``` html
-<div class="parent">
-  <child>
-    <template slot-scope="props">
-      <span>hello from parent</span>
-      <span>{{ props.text }}</span>
-    </template>
-  </child>
-</div>
-```
-
-If we render the above, the output will be:
+Scoped slots allow the definition of dynamic slot templates, using data passed from a child component. For example, in this hypothetical `<review-list>` component:
```

Comment:
I prefer the old text but the new code example

Reply:
We could also use a bit more explanation on the problem:

> Imagine we have a `<review-list>` component that renders a list of reviews. It would be easy if the component simply renders the same markup for each review all the time. However, sometimes we may want to allow other developers using the component to customize the template for each review item.

> We know that we can use slots to inject content from parent to the child, but in this case, each review item relies on data that is only available in the child component, and normal slots only have access to parent scope data.

> This is where scoped slots come into play: scoped slots allow you to inject a block of reusable template into the child component that can receive data provided by the child.